README.md

```diff
@@ -79,19 +79,22 @@ this addon directly.
 treeForAddon(tree) {
   let addon = this.addons.find(addon => addon.name === 'ember-cli-babel'); // find your babel addon
 
-  let options = addon.buildBabelOptions({
-    'ember-cli-babel'
-  })
-
-  return addon.transpileTree(tree, {
+  // optionally build up the configuration
+  let config = {
     'babel': {
       // any babel specific options
      },
 
     'ember-cli-babel': {
       // any ember-cli-babel options
     }
-  });
+  };
+
+  // create the babel options to use elsewhere based on the config above
+  let options = addon.buildBabelOptions(config)
```

Comment:
its a tad confusing, as they appear to be the same example (atleast to someone not familiar)